### PR TITLE
Cypress test case failure test on CI run

### PR DESCRIPTION
**Pull request by seenanair**

### CODE REVIEW

File: cypress/e2e/pages/location.cy.ts [hunk 1]
```diff
@@ -142,15 +142,20 @@ describe('Location', () => {
     });
 
     context('when back button is pressed in browser', () => {
```
Comment (khelwood) [inline cypress/e2e/pages/location.cy.ts:144]: This context says "back button is pressed". Does `cy.visit('/locations/STO-002');` match that description?

File: cypress/e2e/pages/location.cy.ts [hunk 2]
```diff
@@ -142,15 +142,20 @@ describe('Location', () => {
...
-        cy.go('back');
         cy.findByText('STAN-2111').should('exist');
         cy.findByText('STAN-4111').should('exist');
         cy.findByText('STAN-3111').should('not.exist');
       });
     });
 
     context('when refreshing the page', () => {
+      before(() => {
+        cy.reload();
```
Comment (khelwood) [inline cypress/e2e/pages/location.cy.ts:157]: Does this need to reload in both the before() and subsequently in the `it('should...)` block?

File: cypress/e2e/pages/location.cy.ts (hunk 1 above)
Comment (seenanair) [reply cypress/e2e/pages/location.cy.ts:144]: Yes, the following code shows how it was navigated. The intention of this test case is to  ensure 'awaitingLabware' list on display is updated following an add action on an item  (which is already in awaiting storage) and it is reflected on navigating to another page. So, even if it is not an exact back location, it make sense to any other location, I hope.
      _cy.visit('/locations/STO-002');
      cy.findByText('Rack 1 in Freezer 1 in Room 1234').click();
      cy.findByText('Box 1 in Rack 1 in Freezer 1 in Room 1234').click();_

Comment (khelwood) [reply cypress/e2e/pages/location.cy.ts:144]: OK as long as you're happy it's testing what it says :+1:

Comment (seenanair) [reply cypress/e2e/pages/location.cy.ts]: I have changed the test case description to _'when navigating to a previous location or another page'_ . Hope that ids makes it more clear

**Review by khelwood — APPROVED** [review]

**Pull request merged**